test/integration/render-tests/free-camera/terrain/style.json

```diff
@@ -5,6 +5,7 @@
       "height": 512,
       "width": 512,
       "operations": [
+        ["wait"],
```

Comment:
logging offline discussion: this is needed for gl-native (render test waits for all to get loaded and renders only once).